ceno_zkvm/src/instructions/riscv/arith_imm.rs

```diff
@@ -113,7 +113,8 @@ mod test {
         let insn_code = encode_rv32(InsnKind::ADDI, 2, 0, 4, imm_i(3));
         let (raw_witin, lkm) = AddiInstruction::<GoldilocksExt2>::assign_instances(
             &config,
-            cb.cs.num_witin as usize,
+            #[allow(clippy::explicit_auto_deref)]
+            (*cb.cs).num_witin as usize,
```

Comment:
this change seems unrelated and just optional, is it trigger any clippy warning?